src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className='App'>
+      <header className='App-header'>
+        <Display />
+        <ButtonPanel />
+      </header>
+    </div>
+  );
+}
```

Comment:
[Optional]
You can transform it into ES6 form. Like the way, you have done for button and other components.